search/query-features-with-arcade-expression/src/main/java/com/esri/samples/query_features_with_arcade_expression/QueryFeaturesWithArcadeExpressionSample.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2022 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.query_features_with_arcade_expression;
+
+import java.util.HashMap;
+
+import javafx.application.Application;
+import javafx.geometry.Point2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.arcade.ArcadeEvaluator;
+import com.esri.arcgisruntime.arcade.ArcadeExpression;
+import com.esri.arcgisruntime.arcade.ArcadeProfile;
+import com.esri.arcgisruntime.data.ArcGISFeature;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+
+public class QueryFeaturesWithArcadeExpressionSample extends Application {
+
+  private MapView mapView;
+  private ArcGISMap map;
+  private Layer policeBeatsLayer;
+  private final String rpdBeatsLayerName = "RPD Beats  - City_Beats_Border_1128-4500";
+  private final String calloutText = "Crimes in the last 60 days: ";
```

Comment:
I think we can have the text all in the one line here, like the following format:
```suggestion
  private final String calloutText = " crimes in the last 60 days";
```

![image](https://user-images.githubusercontent.com/36415565/181029108-b51a317c-fe2f-4d35-897b-5ef667662616.png)